### PR TITLE
Sync private to public

### PR DESCRIPTION
**Pull request by wirecat**
Hello, this is a result of following our instrunctions on syncing our private repo to the public one during a feature release RC.

### CODE REVIEW

File: ownership.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
```
Comment (djdefi) [inline ownership.yaml:1]: We probably want to exclude this file

Comment (wirecat) [reply ownership.yaml]: Removed.

**Comment by gamefiend** [comment]
Can we ignore the workflows in `.github` for the time being? Other than that it looks good.

**Comment by djdefi** [comment]
Wouldn't that effectively disable CI here? We probably still need the main and Docker build jobs at the very least.

**Comment by wirecat** [comment]
@djdefi & @gamefiend I removed the new additional workflows, but kept any updates to the existing ones.

**Comment by gamefiend** [comment]
> Wouldn't that effectively disable CI here? We probably still need the main and Docker build jobs at the very least.

Sorry, really what I meant was to not add workflows from private to public that don't exist in public. Clarified this with @wirecat in Slack.

File: RELEASING.md [hunk 2]
```diff
@@ -15,11 +15,24 @@ Only repo administrator is allowed to run the release script, otherwise it will
 Prior to making a release,
 
 1. Sync any changes that have been merged to backup-utils-private into this repository.
-1. Go through the list of open pull requests and merge any that are ready for merging.
-1. Go through the list of closed pull requests since the last release and ensure those that should be included in the release notes:
-  - have a "bug", "enhancement" or "feature" label,
-  - have a title that clearly describes the changes in that pull request. Reword if necessary.
-1. Perform a dry run (add `--dry-run` to one of the commands below) and verify the version strings are going to be changed and verify the release notes.
+
```
Comment (gamefiend) [inline RELEASING.md:18]: I'm not sure if we want these changes in the public repo.

Comment (wirecat) [reply RELEASING.md:18]: Should this file be removed from the public repo?

Comment (gamefiend) [reply RELEASING.md:18]: I would vote yes, because we don't make releases from public; we sync it and private and then push the changes out from there.

Comment (wirecat) [reply RELEASING.md]: I went ahead and removed it for now.

**Review by gamefiend — APPROVED** [review]
LGTM, ty @wirecat!

**Review by djdefi — APPROVED** [review]

**Pull request merged**